src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingAdjustmentResource.kt

```diff
@@ -338,6 +328,45 @@ class SentencingAdjustmentResource(private val sentencingAdjustmentService: Sent
     @RequestBody @Valid request: UpdateKeyDateAdjustmentRequest
   ): Unit = sentencingAdjustmentService.updateKeyDateAdjustment(adjustmentId, request)
 
+  @PreAuthorize("hasRole('ROLE_NOMIS_SENTENCING')")
+  @ResponseStatus(HttpStatus.NO_CONTENT)
+  @DeleteMapping("/key-date-adjustments/{adjustmentId}")
+  @Operation(
+    summary = "deletes specific key date adjustment",
+    description = "Requires role NOMIS_SENTENCING. Deletes a key date adjustment by id",
+    responses = [
+      ApiResponse(
+        responseCode = "204",
+        description = "the key date adjustment has been deleted"
```

Comment:
probably worth adding that it has been deleted or it was not found so no deletion took place